### DISCUSSION TITLE
TTS ESPNet into TensorRT or ONNX format

### DISCUSSION THREAD

**Discussion by shahizat** [espnet/espnet#4858] [Q&A] [answered]
Hi, I am trying to convert TTS ESPNet model into ONNX or TensorRT format for edge deployment. Has anyone tried to convert it?Thanks for your reply in advance!

**Comment by b-flo** [comment, marked as answer]
For ONNX, please refer to https://github.com/espnet/espnet_onnx. Also, there are works in progress to add native ONNX support into ESPnet directly.
